lib/resty/etcd.lua

```diff
@@ -89,17 +106,19 @@ function _M.new(opts)
         return nil, 'opts must be table'
     end
 
-    local timeout = opts.timeout or 5000    -- 5 sec
+    local timeout = opts.timeout or 5    -- 5 sec
     local http_host = opts.host or "http://127.0.0.1:2379"
     local ttl = opts.ttl or -1
     local prefix = opts.prefix or "/v2/keys"
+    local user = opts.user or ""
```

Comment:
use empty string as default value is not a good idea. we have to check it by `user == ""`.

Therefore, if not specified, it is best to use `nil`, and we can check it by `if user then`.